src/components/30-organisms/table-sortable/index.js

```diff
@@ -0,0 +1,202 @@
+import { LitElement, css, unsafeCSS, html } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import defineOnce from '../../../utils/define-once';
+import tableCss from './index.scss';
+// TODO use THE NPM package one axa-table is released
+import '../table';
+
+const ASC = 'ascending';
+const DESC = 'descending';
+
+const mapAsc = {
+  [ASC]: 'ASC',
+  [DESC]: 'DESC',
+};
+
```

Comment:
What's the use of this upper block? Is it for documentation, so that people understand what asc and decs means? I would completely get rid of it if possible and just name it ASC and DESC. This const is only used once and I don't see a need for this structure.

Reply:
?? the usage is to map 'ascending' to 'ASC', as i need it lower in the code...

Reply:
cant get rid of it